include/rocksdb/file_system.h

```diff
@@ -86,14 +76,10 @@ struct IOOptions {
   // Timeout for the operation in microseconds
   std::chrono::microseconds timeout;
 
-  // Priority - high or low
-  IOPriority prio;
-
   // Priority used to charge rate limiter configured in file system level (if
```

Comment:
@anand1976 - consider we are now calling this "io_priority" (a more general name), should we revise the comment  "Priority used to charge rate limiter configured in file system level" to be more general?

Reply:
Thinking of something like:
"The priority of each IO, which will be initialized to the value RocksDB decides for that IO based on various factors when passed to the underlying file system level".

Speaking of "decided ... based on various factors", which is basically the table in PR summary of https://github.com/facebook/rocksdb/pull/9996,https://github.com/facebook/rocksdb/pull/9988, I realized users have no way to know that just by looking at any API **in the code**. 

Is it supposed to be known by users? If it is, should we improve the API doc in the code?

Reply:
I agree. Something like "Priority of the IO relative to other RocksDB IO. The file system implementation should consider this when determining the appropriate QoS for the IO request (including, but not limited to, rate limiting)." might be more accurate.

Reply:
> I agree. Something like "Priority of the IO relative to other RocksDB IO. The file system implementation should consider this when determining the appropriate QoS for the IO request (including, but not limited to, rate limiting)." might be more accurate.

Make sense - does the file system implementation need to know how we decide the "Priority of the IO relative to other RocksDB IO" through some kind of API doc?

Reply:
Thanks! I will update the comments.